repos/exporter/lib/handler.ts

```diff
@@ -71,6 +73,30 @@ export const handler: Handler<ExportLambdaBody> = async ({ paths, noc, exportPre
                 passengerType = singleOrGroupPassengerType.passengerType;
             }
 
+            let cappedProductGroupInfo = undefined;
+            if (
+                ticketWithIds.type === 'capped' &&
+                'cappedProductInfo' in ticketWithIds &&
+                'productGroup' in ticketWithIds.cappedProductInfo
+            ) {
+                const productGroupId = ticketWithIds.cappedProductInfo.productGroup.id;
+                const productGroup = await getProductGroupById(noc, productGroupId);
+                const productNames: string[] = [];
+                const matchingJsonLinks = await getProductsById(noc, productGroup.productIds);
+
+                for (const matchingJsonLink of matchingJsonLinks) {
+                    const object = await s3.getObject({ Key: matchingJsonLink, Bucket: PRODUCTS_BUCKET }).promise();
+                    if (!object.Body) {
+                        throw new Error(`body was not present [${path}]`);
```

Comment:
We need to be logging the matchingJsonLink here also, so we know which one didnt work